packages/fetch/src/request-options.interface.ts

```diff
@@ -1,8 +1,9 @@
 import { FetchOptions } from './fetch.interface';
 
-interface ExtraOptions {
+export interface ExtraOptions {
   json?: any;
   headers?: Record<string, string> | Map<string, string>;
+  searchParams?: URLSearchParams | [string, string][] | Record<string, string> | Map<string, string>;
```

Comment:
I know the class is `URLSearchParams` but how would you feel about just calling it `query` instead of `searchParams`? I think this better mirrors comparable libraries